exasol_integration_test_docker_environment/test/test_ssh_access.py

```diff
@@ -1,11 +1,18 @@
+import os
 import pytest
 
-from exasol_integration_test_docker_environment.lib.base.ssh_access import SshKey
+from .conftest import find_container
+from exasol_integration_test_docker_environment.lib.base.ssh_access import SshFiles
 
 
 def test_generate_ssh_key_file(database):
+    test_name = "test_generate_ssh_key_file"
     with database(
+            name=test_name,
             additional_parameters = ["--db-os-access", "SSH"],
     ):
-        file = SshKey.default_folder() / "id_rsa"
-    assert file.exists()
+        files = SshFiles()
+        container = find_container("db_container", test_name)
```

Comment:
with the API you get actually this container name via the EnvironmentInfo which contains a DatabaseInfo

Reply:
If you don't want to use the API, this approach can produce false positives and with that flaky tests, because other test with maybe similar name could leave container behind which might match as well.

Reply:
Switched to ApiTestEnvironment